pkgs/tools/package-management/nix-template/default.nix

```diff
@@ -8,25 +9,35 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "nix-template";
-  version = "0.1.1";
+  version = "0.1.4";
 
   src = fetchFromGitHub {
+    name = "${pname}-${version}-src";
```

Comment:
why this?

Reply:
fetchFromGitHub will name everything as just "source", but passing version number, you're guarenteed to get a different sha256 and avoid stale FOD's